### ISSUE TITLE
Uploading a .PNG throws a invalid URL due to a DNS error.

### ISSUE THREAD

**Issue by suneetsasidharan** [assistx/openbadges#22]
```
Environment - Win8 and Google Chrome
How Found - Microsoft New SDET Bug Bash
Priority - 1
Severity - 1
Description and Customer Impact - Customer is unable to upload his own custom badges. 
Repro Steps - 
• Enter an email address to get badges
• Login using the any of the 4 working login methods
• Click on "Cool, Lets go"
• Go ahead and accept the badges by clicking "Yup"
• Try and upload a custom badge in the next page.


Results - The error message as shown in the attached screenshot
Expected - App should be able to upload the image
```

![Png upload](https://f.cloud.github.com/assets/1211718/260216/30f35558-8cd1-11e2-9eb4-dbb9310777a3.jpg)

**Comment by lilian-wang** [comment]
Duplicate of Issue https://github.com/assistx/openbadges/issues/1

**Comment by msousley** [comment]
this has been fixed in v1.0
